_data/accessibility-tests/identifier.yml

```diff
@@ -0,0 +1,73 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Identifier appears consistently. 
+    summary_additional: The identifier appears in the same location across all pages of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+# Zoom/screen magnification tests
+  - summary: Identifier links are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, identifier links are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Identifier content is functional when screen is resized. 
+    summary_additional: When you zoom in to 200%, other content does not block or overlap the identifier content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Keyboard navigates through links in logical order.
+    summary_additional: When tabbing through the identifier, you can access the links in a logical reading order.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Keyboard navigates through all elements in identifier.
+    summary_additional: When you navigate through the identifer with a keyboard, you can interact with every element inside the identifier.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Focus indicator is clearly visible. 
+    summary_additional: When you use a keyboard to focus on identifier elements, will have a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+  - summary: Identifier does not trap keyboard focus.
+    summary_additional: When you navigate with a keyboard, you can easily move into and out of the identifier.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Focus follows logical content order.
+    summary_additional: When you tab through the identifier with a keyboard, the focus stays in order of the content as intended. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.3
+  - summary: Tabbing alone does not activate a link or other element.
+    summary_additional: When you move around in the identifier with a keyboard, you have to take specific action to activate any interactive elements.
```

Comment:
@mahoneycm I am wondering if this has the same issue as the gherkin for Header?

Reply:
@finekatie It looks like this criteria (3.2.1) is not currently being added to header but it would probably be a good addition. 

[Link to discussion](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?disco=AAABSYuQi6o) in the checklist spreadsheet (🔒)

If that's correct, we can go ahead and resolve this comment since it has to do with the header checklist and not this identifier checklist 👍

If that isn't what you meant, let me know!